package/src/test.rs

```diff
@@ -517,6 +518,26 @@ fn test_dot_env_loading(scie_pants_scie: &Path, clone_root: &TempDir) {
     .unwrap();
 }
 
+fn test_dot_env_error(scie_pants_scie: &Path) {
```

Comment:
I have no idea what to make of exchanges like this: https://github.com/pantsbuild/scie-pants/pull/339#issuecomment-1868017847

That PR got submitted trampling this test. The test seemed like a great idea to me.